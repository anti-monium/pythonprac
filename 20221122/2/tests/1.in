Äåíåã áûëî 1000?
Ïîòîì ñòàëî 12?.
Ìåìíàÿ êîøêà ñêàçàëà ?.
© Áðþñ Óèëëèñ

