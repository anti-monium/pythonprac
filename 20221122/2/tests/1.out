Денег было 1000?
Потом стало 12?.
Мемная кошка сказала ?.
© Брюс Уиллис
